Tests/Form/Type/DocumentTypeTest.php

```diff
@@ -30,7 +30,10 @@ public function setUp()
 
     public function testDocumentManagerOptionSetsEmOption()
     {
-        $field = $this->factory->createNamed('name', 'document', null, array(
+        $typeClass = method_exists('Symfony\Component\Form\AbstractType', 'getBlockPrefix')
+            ? 'Doctrine\Bundle\MongoDBBundle\Form\Type\DocumentType' : 'document';
```

Comment:
Right - so starting with 2.8 forms are created using FQCN? Excuse the silly questions, I still haven't gotten past 2.6 ;)